Peripheral/ch32v20x/inc/ch32v20x_it.h

```diff
@@ -0,0 +1,20 @@
+/********************************** (C) COPYRIGHT *******************************
+ * File Name          : ch32v20x_it.h
```

Comment:
Mhm please remove this because it's usually defined by the firmware project itself, not the SDK. See e.g. https://github.com/Community-PIO-CH32V/platform-ch32v/blob/develop/examples/ble-hid-ch32v208/src/ch32v20x_it.h.